controllers/storagecluster/cephobjectstores.go

```diff
@@ -173,6 +173,7 @@ func (r *StorageClusterReconciler) newCephObjectStoreInstances(initData *ocsv1.S
 					EnableCrushUpdates: true,
 					FailureDomain:      initData.Status.FailureDomain,
 					Replicated:         generateCephReplicatedSpec(initData, "data"),
+					Parameters:         map[string]string{"bulk": "true"},
 				},
 				MetadataPool: cephv1.PoolSpec{
```

Comment:
@BlaineEXE Should we be setting the bulk flag on the object and filesystem metadata pools, in addition to the data pools?

Reply:
Anthony and Kyle seemed to be saying yes when we chatted. It sounds a bit counterintuitive to me as well. My best guess is that the performance gains from increased parallelism must end up being a bigger positive effect than the performance loss from having info split into more chunks.

Reply:
The objects are striped based on rbd order or rgw / cephfs stripe size, so having more PGs doesn't translate to smaller rados objects, it just means we have more targets (pgs) to distribute objects across, and more targets (pgs) to do work.

Reply:
Thanks @mmgaggle for such a detailed analysis.